be/src/common/config.h

```diff
@@ -916,6 +916,10 @@ CONF_String(spill_local_storage_dir, "${STARROCKS_HOME}/spill");
 CONF_mBool(experimental_spill_skip_sync, "true");
 // spill Initial number of partitions
 CONF_mInt32(spill_init_partition, "16");
+// make sure 2^spill_max_partition_level < spill_max_partition_size
+CONF_Int32(spill_max_partition_level, "7");
+CONF_Int32(spill_max_partition_size, "1024");
```

Comment:
check if it is valid when use it?

Reply:
At present, our configure does not have a good verification framework. It usually needs to be checked at startup. We can check them in the future.